### PR TITLE
Fixed bug where is no primary user in payload, would default to false

### PR DESCRIPTION
**Pull request by aaron-russell**
#### Tests
Does this PR include tests for the changes introduced?
- [ ] Yes
- [X] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: server/models/classes/user.js [hunk 1]
```diff
@@ -231,7 +238,7 @@ class User {
         this._password = escape(document.password);
       }
 
-      if (document.isPrimary !== null) {
+      if (this._isBool(document.isPrimary)) {
         // by explicitly checking for "true", don't have to worry about any other value
         if (document.isPrimary === true) {
```
Comment (duncanc19) [inline server/models/classes/user.js:243]: If you've already checked that it's a bool, can't you remove the if else and just assign it to document.isPrimary?

**Review by g-m-taylor — APPROVED** [review]

File: server/models/classes/user.js [hunk 2]
```diff
@@ -217,6 +217,13 @@ class User {
     }
   }
 
+  _isBool(value) {
+    if (typeof value === 'boolean') {
+      return true;
+    }
+    return false;
+  }
```
Comment (duncanc19) [inline server/models/classes/user.js:225]: Could you just do return typeof value === 'boolean'?

File: server/models/classes/user.js (hunk 1 above)
Comment (aaron-russell) [reply server/models/classes/user.js]: Good shout, done!

File: server/models/classes/user.js (hunk 2 above)
Comment (aaron-russell) [reply server/models/classes/user.js:222]: Another good shout, done

**Review by duncanc19 — APPROVED** [review]

**Pull request merged**